src/domain/Models/Enums/IncludesPgce.cs

```diff
@@ -2,8 +2,12 @@ namespace GovUk.Education.SearchAndCompare.Domain.Models.Enums
 {
     public enum IncludesPgce
```

Comment:
We should refactor this to update the names in the news context. Tech debt card...

Reply:
https://trello.com/c/uHJsZLD2/421-rename-includespgce-enum